src/frontend/src/components/LinkEditView.tsx

```diff
@@ -0,0 +1,70 @@
+import { useAllLinkTypes } from '../hooks/projects.hooks';
+import LoadingIndicator from './LoadingIndicator';
+import ErrorPage from '../pages/ErrorPage';
+import { Button, Grid, IconButton, MenuItem, TextField } from '@mui/material';
+import { FieldArrayWithId, UseFieldArrayAppend, UseFieldArrayRemove, UseFormRegister } from 'react-hook-form';
+import DeleteIcon from '@mui/icons-material/Delete';
+import { LinkCreateArgs } from 'shared';
+
+const LinkEditView: React.FC<{
```

Comment:
LinksEditView

Reply:
also general note on this component, it's kinda weird that you can add multiple of a link type, cause then it's just two links with the same label like this. I think we should make it only one of each link type per project
<img width="894" alt="image" src="https://github.com/Northeastern-Electric-Racing/FinishLine/assets/29521172/f89c49b8-6691-4d1c-ab3d-49d564330467">

Reply:
hmmm maybe

Reply:
we gotta fix this problem where u can have two links with the same name (it's confusing), or remove the ability to have more than one link type per project

Reply:
i think wed have to only allow one type per element because idk how wed change the name